src/pyOpenMS/pxds/LocalLinearMap.pxd

```diff
@@ -5,16 +5,30 @@ from Matrix cimport *
 cdef extern from "<OpenMS/ANALYSIS/PIP/LocalLinearMap.h>" namespace "OpenMS":
     
     cdef cppclass LocalLinearMap "OpenMS::LocalLinearMap":
+        # wrap-doc:
+            #   Trained Local Linear Map (LLM) model for peak intensity prediction
+            #   -----
+            #   This class offers a model for predictions of peptide peak heights
+            #   (referred to as intensities) by a Local Linear Map (LLM) model and
+            #   is the basis of PeakIntensityPredictor
+            #   -----
+            #   A general introduction to the Peak Intensity Predictor (PIP)
+            #   can be found in the PIP Tutorial
+            #   -----
+            #   The model trained needs two files for storing the position of the
+            #   codebook vectors and the linear mappings (codebooks.data, linearMapping.data)
+            #   This is the default model used by PeakIntensityPredictor
+
         LocalLinearMap() nogil except +
-        # private
         LocalLinearMap(LocalLinearMap &) nogil except + # wrap-ignore
```

Comment:
Please add the `# private` statement again

Reply:
Then it is clear that the copy constructor is private and why we use # wrap-ignore